File: DevExpress-Demo/VB/MVVMDemo.Wpf/Modules/DXEvent/EventArgsViewModel.vb
Imports DevExpress.Mvvm
Imports System.Windows

Namespace MVVMDemo.DXEventDemo

    Public Class EventArgsViewModel
        Inherits BindableBase

        Public Sub ShowPersonDetail(ByVal person As PersonInfo)
            If person IsNot Nothing Then MessageBox.Show(person.FullName, "Detail info")
        End Sub
    End Class
End Namespace

File: DevExpress-Demo/CS/MVVMDemo.Wpf/Modules/DXEvent/EventArgsViewModel.cs
using DevExpress.Mvvm;
using System.Windows;

namespace MVVMDemo.DXEventDemo {
    public class EventArgsViewModel : BindableBase {
        public void ShowPersonDetail(PersonInfo person) {
            if(person != null)
                MessageBox.Show(person.FullName, "Detail info");
        }
    }
}
